src/get_imports/import_path_visitor.rs

```diff
@@ -0,0 +1,378 @@
+use std::collections::{HashMap, HashSet};
+
+use swc_core::visit::swc_ecma_ast;
+use swc_ecma_ast::{
+    BindingIdent, CallExpr, Callee, Id, ImportDecl, Lit, ModuleExportName, TsImportEqualsDecl,
+};
+use swc_ecmascript::visit::{Visit, VisitWith};
+#[derive(Debug)]
+pub struct ImportPathVisitor {
```

Comment:
You should document what this visitor does, and that it expects the tree to have already resolved w/ a code reference to `swc_core::transforms::resolver`.